backend/main.go

```diff
@@ -29,11 +41,16 @@ func main() {
 		panic(err)
 	}
 
+	if isFlagSet("production") {
```

Comment:
```suggestion
	if *production {
```

I believe you can just do this, and remove the `isFlagSet` function.

Reply:
It's not the same behavior.
With `isFlagSet`  and `--production=false` we can force the production mode to be `false` even if is set to `true` in configuration file.